game-core/src/main/java/games/strategy/engine/data/gameparser/GameParser.java

```diff
@@ -395,149 +371,118 @@ private void parseResources(final Element root) throws GameParseException {
     return players;
   }
 
-  private void parseRelationshipTypes(final Element root) {
-    getChildren("relationshipType", root).stream()
-        .map(e -> e.getAttribute("name"))
+  private void parseRelationshipTypes(final RelationshipTypes relationshipTypes) {
+    relationshipTypes.getRelationshipTypes().stream()
+        .map(RelationshipTypes.RelationshipType::getName)
         .map(name -> new RelationshipType(name, data))
         .forEach(data.getRelationshipTypeList()::addRelationshipType);
   }
 
-  private void parseTerritoryEffects(final Element root) {
-    getChildren("territoryEffect", root).stream()
-        .map(e -> e.getAttribute("name"))
+  private void parseTerritoryEffects(final TerritoryEffectList territoryEffectList) {
+    territoryEffectList.getTerritoryEffects().stream()
+        .map(TerritoryEffectList.TerritoryEffect::getName)
         .forEach(name -> data.getTerritoryEffectList().put(name, new TerritoryEffect(name, data)));
   }
 
-  private void parseUnits(final Element root) {
-    getChildren("unit", root).stream()
-        .map(e -> e.getAttribute("name"))
+  private void parseUnits(final UnitList unitList) {
+    unitList.getUnits().stream()
+        .map(UnitList.Unit::getName)
         .map(name -> new UnitType(name, data))
         .forEach(data.getUnitTypeList()::addUnitType);
   }
 
-  private void parsePlayerList(final Element root) {
-    final PlayerList playerList = data.getPlayerList();
-    for (final Element current : getChildren("player", root)) {
-      final String name = current.getAttribute("name");
-      final boolean isOptional = current.getAttribute("optional").equals("true");
-      final boolean canBeDisabled = current.getAttribute("canBeDisabled").equals("true");
-      final String defaultType = current.getAttribute("defaultType");
-      final boolean isHidden = current.getAttribute("isHidden").equals("true");
-      final GamePlayer newPlayer =
-          new GamePlayer(name, isOptional, canBeDisabled, defaultType, isHidden, data);
-      playerList.addPlayerId(newPlayer);
-    }
-  }
-
-  private void parseAlliances(final Element root) throws GameParseException {
+  private void parsePlayerList(final org.triplea.map.data.elements.PlayerList playerListData) {
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'org.triplea.map.data.elements.PlayerList' due to existing import 'org.triplea.map.data.elements.PlayerList'](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6154389)